servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcClientBuilder.java

```diff
@@ -50,72 +58,235 @@
+     * data and log only network events.
+     * @return {@code this}.
+     */
     public abstract GrpcClientBuilder<U, R> enableWireLogging(String loggerName, LogLevel logLevel,
                                                               BooleanSupplier logUserData);
 
-    @Override
+    /**
+     * Configurations of various underlying protocol versions.
+     * <p>
+     * <b>Note:</b> the order of specified protocols will reflect on priorities for ALPN in case the connections are
+     * secured.
+     *
+     * @param protocols {@link HttpProtocolConfig} for each protocol that should be supported.
+     * @return {@code this}.
+     */
     public abstract GrpcClientBuilder<U, R> protocols(HttpProtocolConfig... protocols);
 
-    @Override
+    /**
+     * Set default timeout during which gRPC calls are expected to complete. This default will be used only if the
+     * request metadata includes no timeout; any value specified in client request will supersede this default.
+     *
+     * @param defaultTimeout {@link Duration} of default timeout which must be positive non-zero.
+     * @return {@code this}.
+     */
     public abstract GrpcClientBuilder<U, R> defaultTimeout(Duration defaultTimeout);
 
-    @Override
+    /**
+     * Append the filter to the chain of filters used to decorate the {@link ConnectionFactory} used by this
+     * builder.
+     * <p>
+     * Filtering allows you to wrap a {@link ConnectionFactory} and modify behavior of
+     * {@link ConnectionFactory#newConnection(Object, TransportObserver)}.
+     * Some potential candidates for filtering include logging and metrics.
+     * <p>
+     * The order of execution of these filters are in order of append. If 3 filters are added as follows:
+     * <pre>
+     *     builder.append(filter1).append(filter2).append(filter3)
+     * </pre>
+     * Calling {@link ConnectionFactory} wrapped by this filter chain, the order of invocation of these filters will be:
+     * <pre>
+     *     filter1 ⇒ filter2 ⇒ filter3 ⇒ original connection factory
+     * </pre>
+     * @param factory {@link ConnectionFactoryFilter} to use.
+     * @return {@code this}.
+     */
     public abstract GrpcClientBuilder<U, R> appendConnectionFactoryFilter(
             ConnectionFactoryFilter<R, FilterableStreamingHttpConnection> factory);
 
-    @Override
+    /**
+     * Append the filter to the chain of filters used to decorate the {@link StreamingHttpConnection} created by this
+     * builder.
+     * <p>
+     * Filtering allows you to wrap a {@link StreamingHttpConnection} and modify behavior during request/response
+     * processing
+     * Some potential candidates for filtering include logging, metrics, and decorating responses.
+     * <p>
+     * The order of execution of these filters are in order of append. If 3 filters are added as follows:
+     * <pre>
+     *     builder.append(filter1).append(filter2).append(filter3)
```

Comment:
To be consistent with HTTP, please use the real method name here: `appendConnectionFilter`